src/pages/components/table-of-contents.mdx

```diff
@@ -32,7 +32,16 @@ The Table of Contents (ToC) component renders as left navigation that includes a
 
 <Title>Use case</Title>
 
-![Image of Table of contents component](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+![Image of Table of contents component usecase](../../images/component/table-of-contents/table-of-content-default-use-case.jpg)
+
+**When to use:**
+* When the content of the page is long or text heavy and will require a lot of scrolling.
+* When users need to know what is included in a page, the table of contents can serve as an outline or summary of the page content.
+* When users need to navigate to a specific section on the page quickly.
+
+**When not to use:**
+* When a page is relatively short and the user will not have to scroll very far.
```

Comment:
Could we get any more specific with what is meant by `relatively short/long`?

Reply:
@photodow Is this working any better? "When a page content is short, the user does not need to rely on page scrolling to view the information."

Reply:
I guess to be more specific I wonder if we could provide something measurable. Like if the user has to scroll more than 5 times, or the content is generally 2x the window size, or more than 4 titled sections on the page. I'm not saying these are the things, but wondering if it makes sense to give them something to aim for. A ballpark figure if that makes sense. I assume our content model would have a minimum amount of ToC items that could even be used to start with.

Reply:
> I assume our content model would have a minimum amount of ToC items that could even be used to start with.

Agreed we should provide some guidelines on minimum and recommended max (we won't limit).
I wasn't sure if we would include that sort of detail in the content model section, like what Michael has been working on. 

> I wonder if we could provide something measurable. Like if the user has to scroll more than 5 times, or the content is generally 2x the window size, or more than 4 titled sections on the page.

I was struggling on how to phrase this guideline because in my opinion, maybe because it's a bit of a gray area..

The partners page is one I would not recommend using a TOC, because it's a series of short bands, and the TOC won't really do much in terms of scrolling down the page. 
![screencapture-ibm-partners-2021-04-02-18_04_54](https://user-images.githubusercontent.com/45692622/113457737-906cbc00-93de-11eb-8235-f6d0aebfe7ff.png)

The NOT using with short sections is partly why I had phrased it as "when the content of the page is long or text heavy" to try and describe what kind of content would use a TOC.. but maybe it's not quite right yet.